frontend/src/components/Admin/CoffeeChats/CoffeeChats.tsx

```diff
@@ -127,11 +127,11 @@ const CoffeeChats: React.FC = () => {
               placeholder={'View Member Bingo Board'}
               fluid
               selection
-              value={selectedMember ? selectedMember.netid : undefined}
+              value={selectedMember ? selectedMember.netid : 'View Member Bingo Board'}
```

Comment:
nit: for the sake of not duplicating this text with the placeholder, could you define it as a const above, like `DEFAULT_MEMBER_DROPDOWN_TEXT`